test/UnitTests/TestFramework.UnitTests/Assertions/CollectionAssertTests.cs

```diff
@@ -198,6 +198,13 @@ public void CollectionAssertAreEqualComparerNullabilityPostConditions()
         comparer.ToString(); // no warning
     }
 
+    public void CollectionAssertAreEqual_WithIgnoreCaseComparer_DoesNotThrow()
```

Comment:
Do we have a case similar to this one but where we would use a `StringComparer` that doesn't ignroe casing?